docker/volumeplugin/rancher.go

```diff
@@ -108,6 +110,24 @@ func isCreated(driver string, vol client.Volume) bool {
 	return goodStates[vol.State]
 }
 
+func (r *RancherState) GetAny(name string) (*volume.Volume, *client.Volume, error) {
+	vols, err := r.client.Volume.List(&client.ListOpts{
```

Comment:
I notice that both Get and GetAny don't check for the storageDriverId, They should.